picoquic/logwriter.c

```diff
@@ -852,8 +794,15 @@ void picoquic_cc_dump(picoquic_cnx_t * cnx, uint64_t current_time)
 
     bytewrite_int32(ps_head, (uint32_t)bytestream_length(ps_msg));
 
-    (void)fwrite(bytestream_data(ps_head), bytestream_length(ps_head), 1, cnx->cc_log);
-    (void)fwrite(bytestream_data(ps_msg), bytestream_length(ps_msg), 1, cnx->cc_log);
+    if (cnx->cc_log != NULL) {
```

Comment:
Since the code for opening a cc_log has been removed, is it still possible to come here?

Reply:
Let me check. The updated code writes the congestion control reports in the "binlog" frame. I have verified that doing that I can extract the same CSV files -- the packet trace test passes. The intent is to document congestion control events in the qlog, but I have not added that code yet.